src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3341,6 +3363,25 @@ private int GetOrCreatePlaceholderSlot(object identifier, TypeWithAnnotations ty
             return GetOrCreateSlot(placeholder, forceSlotEvenIfEmpty: true);
         }
 
+        private bool TryGetPlaceholderSlot(BoundExpression node, out int slot)
```

Comment:
>BoundExpression [](http://example.com/codeflow?start=43&length=15)

It looks like this function is only used for BoundInterpolatedStringArgumentPlaceholder. I think we can benefit from making the parameter type stronger.

Reply:
This function is being removed.